package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "async": "^1.5.2",
     "cookie-parser": "^1.3.2",
-    "debug": "^2.2.0",
+    "debug": "^2.6.9",
```

Comment:
This is effectively a no-op, because `^2.2.0` will install the latest 2.x version, e.g. `2.6.9.` Did you perhaps mean `^3.1.0`?